package.json

```diff
@@ -12,9 +12,18 @@
     ]
   },
   "dependencies": {
-    "lerna": "3.16.4"
+    "got": "10.2.1",
+    "lerna": "3.16.4",
+    "next": "9.1.7",
+    "parse-github-url": "1.0.2",
+    "react": "16.12.0",
+    "react-dom": "16.12.0",
+    "tar-fs": "2.0.0",
+    "unzip-stream": "0.3.0"
```

Comment:
Can you revert these changes and instead create `/api/package.json` with the necessary deps?

Reply:
@styfle we need it in the root in order to make Next.js work though, don't we

Reply:
@AndyBitz I don't think so. Furthermore, why are we using Next.js if this is only an API? Are we expecting to build a frontend in the future?

Reply:
@styfle likely, also, we need to build the root, because we need `packages/frameworks` as well, if we'd put it inside a subdirectory we wouldn't be able to.

Reply:
Why not? I believe that node-file-trace will read `packages/frameworks` properly if you import `@now/frameworks` from the api dir.

Reply:
I had to use `../../packages/frameworks` in order to make it work, probably because `api` is not a package for `lerna` (not sure if it should be)?

Reply:
I think `../../packages/frameworks` is acceptable, as long as TS can figure it out 👍